nixos/modules/system/boot/networkd.nix

```
@@ -2013,6 +2023,16 @@ let
       '';
     };
 
+    ipv6PREF64Prefixes = mkOption {
+      default = [];
+      example = [ { Prefix = "2001:db8::/96"; } ];
```

```suggestion
      example = [ { Prefix = "64:ff9b::/96"; } ];
```

https://www.rfc-editor.org/rfc/rfc6052

---

I don't mind either way, but we have a few other places in nixpkgs where NAT64-related things are being done, and we use the rfc3849 documentation prefix (albeit with varying prefix sizes) for those. I guess it's a matter of how the examples are interpreted, as an example of how a valid config looks, or as an example of a config that would be used in the real world. For consistency, I think keeping it as is is preferred.